js-chat/tests/message.test.ts

```diff
@@ -122,16 +146,16 @@ describe("Send message test", () => {
     expect(deletedMessage).toBeUndefined()
   }, 30000)
 
-  test("should restore a soft deleted message", async () => {
+  test("should soft delete message", async () => {
     await channel.sendText("Test message")
-    await sleep(150) // history calls have around 130ms of cache time
```

Comment:
Why in some case there is 

>  // History calls have around 130ms of cache time

and in some you deleted this comment e.g. line 134 ?